### PR TITLE
Updated BUILDING_ON_WINDOWS.md

### PR DESCRIPTION
**Pull request by cinemassacres**
Pull request checklist:

- [ ] `CHANGELOG.md` was updated, if applicable

# Description

fixed the link going to Openssl it was g but they moved to h now

### CODE REVIEW

**Comment by fourtf** [comment]
Lgtm

**Comment by fourtf** [comment]
This tells me that it is "This branch is out of sync with the base branch." So I can't merge. Not sure why this happens.

File: BUILDING_ON_WINDOWS.md [hunk 1]
```diff
@@ -29,7 +29,7 @@ Note: This installation will take about 1.5 GB of disk space.
 
 ## OpenSSL
 ### For our websocket library, we need OpenSSL 1.1
-1. Download OpenSSL for windows, version `1.1.1g`: **[Download](https://slproweb.com/download/Win64OpenSSL-1_1_1g.exe)**
+1. Download OpenSSL for windows, version `1.1.1g`: **[Download](https://slproweb.com/download/Win64OpenSSL-1_1_1h.exe)**
```
Comment (TranRed) [inline BUILDING_ON_WINDOWS.md:32]: the text should also be changed here, not only the download link.

**Review by TranRed — CHANGES_REQUESTED** [review]

File: BUILDING_ON_WINDOWS.md (hunk 1 above)
Comment (fourtf) [reply BUILDING_ON_WINDOWS.md:32]: trueeeee

**Pull request merged**